packages/app/src/cli/services/app-logs/poll-app-logs.ts

```diff
@@ -52,19 +62,23 @@ export const pollAppLogs = async ({
       if (response.status === 401) {
         await resubscribeCallback()
       } else if (response.status === 429 || response.status >= 500) {
-        stdout.write(`Received an error while polling for app logs.`)
-        stdout.write(`${response.status}: ${response.statusText}`)
-        stdout.write(responseText)
-        stdout.write(`Retrying in ${POLLING_BACKOFF_INTERVAL_MS / 1000} seconds`)
+        outputDebug(`Received an error while polling for app logs.`)
+        outputDebug(`${response.status}: ${response.statusText}`)
+        outputDebug(responseText)
+        outputDebug(`Retrying in ${POLLING_BACKOFF_INTERVAL_MS / 1000} seconds`)
+
```

Comment:
TDB: Either this needs to be something both `logs` and `dev` command can call. (`stdout.write` will display output correctly for both the `Dev` and `Logs` components).

OR

(I think prefered) do not use the stdout for this.